socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,28 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
         for a_store in self.stores.itervalues():
-            a_store.save_raw_and_processed(
-              raw_crash,
-              dump,
-              processed_crash,
-              crash_id
+            store_name = getattr(
+                a_store, 'wrapped_object', a_store.__class__
             )
+            # self.logger.debug('writing %s to %s' % (crash_id, store_name))
+
+            self.quit_check()
+            try:
+                a_store.save_raw_and_processed(
+                    raw_crash,
+                    dump,
+                    processed_crash,
+                    crash_id
+                )
+            except Exception:
+                self.logger.error(
+                    '%s failure: %s', store_name, exc_info=True
```

Comment:
Yes, I do want to show the representation. I'm going to change the variable name to say ``repr`` instead of ``name``.